SwiftRustIntegrationTestRunner/SwiftRustIntegrationTestRunner/Vec.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  String.swift
+//  SwiftRustIntegrationTestRunner
+//
+//  Created by Frankie Nwafili on 2/18/22.
+//
```

Comment:
You can delete this comment. Xcode typically auto generates them but we don't really need them